modules/ROOT/pages/clauses/match.adoc

```diff
@@ -262,151 +228,262 @@ RETURN a, b
 2+|Rows: 2
 |===
 
+[[match-on-relationship-type]]
+=== Filter on relationship types
 
-[[match-on-rel-type]]
-=== Match on relationship type
-
-When the relationship type to match on is known, it is possible to specify it by using a colon (`:`) before the relationship type.
+It is possible to specify the type of a relationship in a relationship pattern by using a colon (`:`) before the relationship type.
 
-.Query
-[source, cypher, indent=0]
+.Relationship pattern filtering on the `ACTED_IN` relationship type
+[source, cypher]
 ----
-MATCH (wallstreet:Movie {title: 'Wall Street'})<-[:ACTED_IN]-(actor)
-RETURN actor.name
+MATCH (:Movie {title: 'Wall Street'})<-[:ACTED_IN]-(actor:Person)
+RETURN actor.name AS actor
 ----
 
-Returns all actors who `ACTED_IN` the movie `Wall Street`.
-
 .Result
-[role="queryresult",options="header,footer",cols="1*<m"]
+[source, role="queryresult",options="header,footer",cols="1*<m"]
 |===
-| actor.name
+| actor
 | "Michael Douglas"
 | "Martin Sheen"
 | "Charlie Sheen"
 |Rows: 3
 |===
 
-Read more about xref:patterns/fixed-length-patterns.adoc#relationship-patterns[relationship type expressions].
-
-[[match-on-multiple-rel-types]]
-=== Match on multiple relationship types
+[[match-on-multiple-relationship-types]]
+=== Filter on multiple relationship types
```

Comment:
for nodes, the new heading is "MATCH using label expressions"
makes me wonder if the two could be unified, "MATCH using node label expressions" and "MATCH using relationship label expressions" or something similar

Reply:
Yep - will do relationship type expressions